apps/account-api/src/controllers/v1/accounts-v1.controller.ts

```diff
@@ -119,7 +119,9 @@ export class AccountsControllerV1 {
     @Param('accountId') accountId: string,
   ): Promise<{ signerPayload: SignerPayloadRaw; encodedPayload: string }> {
```

Comment:
This should be a defined type somewhere; I see it's used in other places, shouldn't have to write it out everywhere we use it. Also a common type definition gives us compile-time type-safety checks.

Reply:
Meant to do that and forgot in everything else. thanks for the catch.